### ISSUE TITLE
_structural.pyd is being copied to the wrong directory

### ISSUE THREAD

**Issue by hsauro** [sys-bio/Libstructural#8]
When building ligstructural, _structural.pyd is currently being copied to BUILD_x64/SWIG/Release.

Shouldn't it be copied to 

INSALL_x64/wrapper/structural

?

**Comment by yosefmaru** [comment]
Yes. It copies in both  BUILD_x64/SWIG/Release and INSALL_x64/wrapper/structural if you target your Build to INSTALL. If you build with ALL_BUILD target, you won't get _structural.pyd in INSALL_x64/wrapper/structural.

**Comment by kirichoi** [comment]
ALL_BUILD should move pyd file to install directory as well.
